.github/workflows/publish.yml

```
@@ -0,0 +1,25 @@
+name: Publish maplibre_gl packages to pub.dev
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

The regex is invalid, could be that GitHub can parse it but we could change it to:

```suggestion
      - 'v\d+\.\d+\.\d+(-\w+)?'
```

This disallows tags like `v00000` and `v0.0.0 some text`

---

Seems like GitHub doesn't use a real regex syntax here? ([docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags))
My bad, didn't knew that.